tests/integration/test_ses.py

```diff
@@ -87,18 +89,25 @@ def test_send_email_save(self, ses_client):
                 "ToAddresses": ["success@example.com"],
             },
         )
+        message_id = message["MessageId"]
 
-        with open(os.path.join(data_dir, "ses", message["MessageId"] + ".json"), "r") as f:
+        with open(os.path.join(data_dir, "ses", message_id + ".json"), "r") as f:
             message = f.read()
 
         contents = json.loads(message)
 
         assert email == contents["Source"]
         assert "A_SUBJECT" == contents["Subject"]
         assert "A_MESSAGE" == contents["Body"]
-        assert ["success@example.com"] == contents["Destinations"]["ToAddresses"]
+        assert ["success@example.com"] == contents["Destination"]["ToAddresses"]
 
-    def test_send_templated_email_save(self, ses_client, create_template):
+        api_contents = requests.get("http://localhost:4566/_localstack/ses").json()
```

Comment:
nit: Let's better use `config.get_edge_url(..)` here, instead of hardcoding the URL and port.

Reply:
Removed all hardcoded values :+1: